<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebAppActionEditor</class>
 <widget class="QWidget" name="WebAppActionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KTabWidget" name="actionTriggerTab">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="metaTab">
      <attribute name="title">
       <string>Meta Information</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;b&gt;Meta Information&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labellabel">
         <property name="text">
          <string>Label</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="label"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Icon</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KIconButton" name="icon">
         <property name="icon">
          <string>internet-web-browser</string>
         </property>
         <property name="iconSize">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Plugin name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="pluginName"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Author</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="author"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="triggerTab">
      <attribute name="title">
       <string>Actions and Triggers</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&lt;b&gt;Action Configuration&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Show On URL</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="showOnUrlLine"/>
       </item>
       <item row="2" column="1">
        <widget class="KListWidget" name="showOnUrl"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Show on Wildcard</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KLineEdit" name="showOnWildcardLine"/>
       </item>
       <item row="4" column="1">
        <widget class="KListWidget" name="showOnWildcard"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Trigger on URL</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="KLineEdit" name="triggerOnUrlLine"/>
       </item>
       <item row="6" column="1">
        <widget class="KListWidget" name="triggerOnUrl"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Trigger on Wildcard</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="KLineEdit" name="triggerOnWildcardLine"/>
       </item>
       <item row="8" column="1">
        <widget class="KListWidget" name="triggerOnWildcard"/>
       </item>
      </layout>
      <zorder>showOnUrl</zorder>
      <zorder>label_7</zorder>
      <zorder>showOnWildcard</zorder>
      <zorder>label_14</zorder>
      <zorder>triggerOnWildcardLine</zorder>
      <zorder>triggerOnWildcard</zorder>
      <zorder>showOnUrlLine</zorder>
      <zorder>triggerOnUrlLine</zorder>
      <zorder>label_9</zorder>
      <zorder>triggerOnUrl</zorder>
      <zorder>label_8</zorder>
      <zorder>showOnWildcardLine</zorder>
      <zorder>label_2</zorder>
      <zorder>label_7</zorder>
      <zorder>showOnUrl</zorder>
      <zorder>showOnWildcard</zorder>
      <zorder>triggerOnWildcardLine</zorder>
      <zorder>label_14</zorder>
      <zorder>triggerOnWildcard</zorder>
      <zorder>showOnUrlLine</zorder>
      <zorder>triggerOnUrlLine</zorder>
      <zorder>triggerOnUrl</zorder>
      <zorder>label_9</zorder>
      <zorder>label_8</zorder>
      <zorder>label_2</zorder>
      <zorder>showOnWildcardLine</zorder>
     </widget>
     <widget class="QWidget" name="scriptsTab">
      <attribute name="title">
       <string>Scripts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="KPushButton" name="kpushbutton">
           <property name="text">
            <string>Save Script</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="saveButton">
       <property name="text">
        <string>Save Action</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
